### PR TITLE
[NVIDIA] Add TRT conversion scripts through ONNX , TRT engine DS deployment script, and some clean up. 

### PR DESCRIPTION
**Pull request by mengdong**
Done:
1, add DS config for TRT file
2, use TRT engine with the config to run a DS example

To do:
Add Deepstream tracker for VOT

### CODE REVIEW

**Comment by mengdong** [comment]
@googlebot I fixed it.

**Comment by ijovicic-nv** [comment]
@googlebot I consent.

**Comment by mengdong** [comment]
@yongzhe2160 could you please review?

File: README.md [hunk 1]
```diff
@@ -99,6 +108,22 @@ pip3 install numpy
...
+First convert the model to ONNX format, use the model downloaded from GCP Video Intelligence console:
+`python3 -m tf2onnx.convert --graphdef frozen_inference_graph.pb --inputs image_tensor:0 --outputs detection_boxes:0,detection_classes:0,detection_scores:0,num_detections:0 --output models/model_opset_10.onnx --fold_const --opset 10 --verbose`
+
+then, 
+`python3 examples/modify_onnx_model_opset10.py --onnx model_opset_10.onnx --modified model_opset_10_modified.onnx`
+
+Then convert ONNX graph to a TensorRT Engine file, please note the TensorRT conversion is a memory intensive process so be mindful when running on less powerful device like Nano:
+`/usr/src/tensorrt/bin/trtexec --onnx=model_opset_10_modified.onnx --fp16 --workspace=2048 --allowGPUFallback --saveEngine=vot_fp16.plan --verbose &> trtexec_build.log`
+
+The log could be viewed by `tail -f trtexec_build.log`. The TensorRT Engine file `vot_fp16.engine` could be deployed to a Deep Stream application.
```
Comment (yongzhe2160) [inline README.md]: should this be vot_fp16.plan, instead of .engine from the command above?

File: deepstream/vot/config.pbtxt.tf [hunk 2]
```diff
@@ -8,25 +8,25 @@ input [
...
+    dims: [ 256,256, 3 ]
   }
 ]
 output [
   {
     name: "detection_boxes"
     data_type: TYPE_FP32
-    dims: [ 40, 4]
-    reshape { shape: [40, 4] }
+    dims: [ 100, 4]
```
Comment (yongzhe2160) [inline deepstream/vot/config.pbtxt.tf:18]: which model was this?

**Review by yongzhe2160 — COMMENTED** [review]
Thanks Dong!

File: README.md (hunk 1 above)
Comment (mengdong) [reply README.md]: good catch, fixed...

File: deepstream/vot/config.pbtxt.tf (hunk 2 above)
Comment (mengdong) [reply deepstream/vot/config.pbtxt.tf:18]: this is the VOT model we downloaded from Cloud console

**Review by yongzhe2160 — APPROVED** [review]

**Pull request merged**